src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -1297,7 +1297,7 @@ QString RekordboxFeature::formatRootViewHtml() const {
             << tr("Beatgrids")
             << tr("Hot cues")
             << tr("Memory cues")
-            << tr("Loops (not yet supported in Mixxx 2.3)");
+            << tr("Loops (currently only the first loop)");
```

Comment:
Well, AFAIK it still imports all loops, but only the first is usable right now. In Mixxx 2.4, the imported loops will become usable without reimporting Unsure if we should state that here though or if that's just confusing.

Reply:
Ok, please correct it however you like or tell me what I should write.

Reply:
```suggestion
            << tr("Loops (only the first loop is currently usable in Mixxx)");
```